examples/cpp/pyperf/PyPerfNativeStackTrace.cc

```diff
@@ -8,28 +8,36 @@
 #include <sys/uio.h>
 #include <errno.h>
 #include <unistd.h>
-
 #include <cstdio>
 #include <cstring>
 #include <sstream>
+#include <time.h>     
+#include <chrono>
+#include <thread>
 
 #include "PyPerfLoggingHelper.h"
 
 namespace ebpf {
 namespace pyperf {
 
 // Ideally it was preferable to save this as the context in libunwind accessors, but it's already used by UPT
+const uint16_t NativeStackTrace::CacheMaxSizeMB = 56;
+const uint16_t NativeStackTrace::CacheMaxTTL = 300; // In seconds
 const uint8_t *NativeStackTrace::stack = NULL;
 size_t NativeStackTrace::stack_len = 0;
 uintptr_t NativeStackTrace::sp = 0;
 uintptr_t NativeStackTrace::ip = 0;
+time_t  NativeStackTrace::now;
```

Comment:
```suggestion
time_t NativeStackTrace::now;
```

Reply:
There some other formatting issues - you can run `./scripts/git-clang-format` to fix them all :)

Reply:
The last commit contains  `git-clang-format` formatting changes for files modified in the PR.